core/src/test/java/edu/ucr/cs/riple/core/tools/CoreTestHelper.java

```diff
@@ -122,6 +129,29 @@ public CoreTestHelper requestCompleteLoop() {
     return this;
   }
 
+  public CoreTestHelper enableDownstreamDependencyAnalysis(
+      String libraryModelLoaderPath, String... dependencies) {
+    // Path to update library model
+    String root =
+        Utility.changeDirCommand(Paths.get(System.getProperty("user.dir")).getParent().toString());
+    String reloadLibraryModel =
+        Utility.changeDirCommand(root) + " && ./gradlew library-model-loader:jar";
+    this.downstreamDependencyAnalysisActivated = true;
+    this.libraryModelLoaderPath = libraryModelLoaderPath;
+    this.downstreamBuildCommands =
+        Arrays.stream(dependencies)
+            .map(
+                projectName ->
+                    reloadLibraryModel
+                        + " && "
+                        + Utility.changeDirCommand(projectPath.toString())
+                        + " && ./gradlew :"
+                        + projectName
+                        + ":compileJava")
```

Comment:
This kind of structured string is a good candidate for `String.format(...)`

Reply:
39f814ee402eb78954a0a454abedb105907e2ab1